### ISSUE TITLE
Question about upgrading d-drl-coordination to multi-agent drl

### ISSUE THREAD

**Issue by burnCalories** [RealVNF/distributed-drl-coordination#6]
hi @stefanbschneider 
Recently, I have finally completed the task of migrating `d-drl-coordination SB3` to the `rllib` version. After adding the curiosity module, I found that a similar success rate can be achieved even with rewards of success and failure.  

I have some interesting ideas.  `d-drl-coordination` is coordinated based on flows with different arrival times. If DRL is upgraded to MARL, for example, three to four agents can process flows in parallel. There may be a higher success rate in MMPP mode or det real world trace mode.

After almost searching the documentation and community of rllib, I found that there is very little information on how to modify a custom environment to a multi-agent environment in it. Could you help me providing some information about this?  :)

I would be happy to share my project with you. However, I have been quite busy recently. Once I have time to upload the complete project, I will let you know as soon as possible.  :)

**Comment by burnCalories** [comment]
Due to network issues, I created two identical issues in a row. Please help me close this issue.

**Comment by stefanbschneider** [comment]
Duplicate of https://github.com/RealVNF/distributed-drl-coordination/issues/5
